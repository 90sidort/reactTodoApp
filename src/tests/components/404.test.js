import NotFoundPage from '../../components/404'
import React from 'react'
import { shallow } from 'enzyme'

test('Should correctly render 404 component', () => {
    const wrapper = shallow(<NotFoundPage />)
    expect(wrapper).toMatchSnapshot()
})