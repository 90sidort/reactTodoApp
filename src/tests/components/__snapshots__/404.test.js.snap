// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should correctly render 404 component 1`] = `
<div>
  404! 
  <Link
    replace={false}
    to="/"
  >
    Go to Dashboard
  </Link>
</div>
`;
